backend/webapi/Features/AccessRequests/HcimAccountTransfer.cs

```diff
@@ -75,14 +75,19 @@ public class CommandHandler : ICommandHandler<Command, IDomainResult<Model>>
 
         public async Task<IDomainResult<Model>> HandleAsync(Command command)
         {
+            var userId = await this.context.Credentials
+            .Where(credential => credential.PartyId == command.PartyId)
+            .Select(credential => credential.UserId)
+            .SingleAsync();
```

Comment:
As is, this will beak as soon as a user has multiple credentials; `Single()` will throw.
I believe the plan is to assign the role to each credential the user has.